### PR TITLE
Migrate to sugardough.

### PR DESCRIPTION
**Pull request by glogiotatidis**
This rebases lumbergh on top of sugardough, upgrades to Django 1.8, changes South migrations to Django migrations.

@bensternthal I figured out that the license for this project is [BSD-3)(https://github.com/mozilla/lumbergh/blob/master/contribute.json#L6). Any reason why this is not MPL? /cc @Osmose 

Also @bensternthal we use both GoogleTagManager and Optimizely. Do we need any of them?

@jgmize I still need to figure out whether we need GTM and/or Optimizely and fix CSP accordingly. Other than that this is ready for r?

TODOs 
 - [x] CSP for GTM / Optimizelly

Demo server:
 - http://careers-stage.us-west.moz.works/#teams

### CODE REVIEW

**Comment by bensternthal** [comment]
Checking on GTM and Optimizely. GTM is likely a yes.. optimizely.. i suspect not.

Will comment here.

**Comment by bensternthal** [comment]
Optimizely: Remove
GTM + GA: We need em both, this was the first site Gareth worked on with GTM and its not fully implemented, so we need them both for now.

**Comment by jgmize** [comment]
@glogiotatidis I took a very brief look at this and your approach seems sane, but this needs more of a review than I currently have the bandwidth to do, and frankly I'm overbooked for the rest of the year. @Osmose is there any chance you'd be willing to review this as favor to me?

**Comment by Osmose** [comment]
I can add it to my list but I too am more or less booked with stuff that needs to get done first for a while.

> Any reason why this is not MPL?

Because Playdoh was BSD, I believe. I usually go Apache now that we have the choice between that and MPL.

**Comment by jgmize** [comment]
@glogiotatidis adding a full set of selenium integration tests may be out of scope for this, but perhaps we should add a minimal set of acceptance tests that at least validate status codes against the deployed instance?

@stephendonner is there any chance that someone on your team could help us out with some manual testing against https://careers-stage.us-west.moz.works?

**Comment by glogiotatidis** [comment]
> @glogiotatidis adding a full set of selenium integration tests may be out of scope for this, but perhaps we should add a minimal set of acceptance tests that at least validate status codes against the deployed instance?

Good point. I filed [bug 1226515](https://bugzilla.mozilla.org/show_bug.cgi?id=1226515) that captures that so we don't block this PR.

**Comment by glogiotatidis** [comment]
> Optimizely: Remove
> GTM + GA: We need em both, this was the first site Gareth worked on with GTM and its not fully implemented, so we need them both for now.

done

File: careers/university/tests/test_views.py [hunk 1]
```diff
@@ -41,25 +41,25 @@ def test_event_filter(self):
...
 
-    def test_open_for_applications(self):
-        """
-        If there are positions in the Intern category,
-        open_for_applications should be True.
-        """
-        JobvitePositionFactory.create(job_type='Intern')
-        response, context = self._index()
-        ok_(context['open_for_applications'])
+    # def test_open_for_applications(self):
```
Comment (Osmose) [inline careers/university/tests/test_views.py]: Mistakenly left in?

**Comment by Osmose** [comment]
I also skimmed this and think it looks good enough to try. I don't have time for a full review but given the limited site functionality this is probably fine to just manually test and deploy given that two people skimmed it and liked it. :P

If you still want/need a deeper review I can leave this on my todo list but it'll be a while.

**Comment by glogiotatidis** [comment]
> I can add it to my list but I too am more or less booked with stuff that needs to get done first for a while.
> 
>     Any reason why this is not MPL?
> 
> Because Playdoh was BSD, I believe. I usually go Apache now that we have the choice between that and MPL.

Given that BSD is super permissive to do whatever and MPL is a proper Free Software license I'm going to change this to MPL

File: careers/university/tests/test_views.py (hunk 1 above)
Comment (glogiotatidis) [inline careers/university/tests/test_views.py]: Mistakenly left commented out. Nice catch!

**Comment by glogiotatidis** [comment]
> I also skimmed this and think it looks good enough to try. I don't have time for a full review but given the limited site functionality this is probably fine to just manually test and deploy given that two people skimmed it and liked it. :P
> 
> If you still want/need a deeper review I can leave this on my todo list but it'll be a while.

wfm! I've tested this and it works, I just wanted a second pair of eyes to check if there is something really broken.

**Comment by glogiotatidis** [comment]
Merged in sugardough branch https://github.com/mozilla/lumbergh/tree/sugardough

Thanks all!

**Comment by Osmose** [comment]
> BSD is super permissive to do whatever and MPL is a proper Free Software license

**Pull request closed without merging**